examples/combobox_liststore/main.rs

```diff
@@ -0,0 +1,100 @@
+use glib::clone;
+use gtk::{
+    prelude::*, CellRendererText, CellRendererToggle, ComboBox, Entry, ListStore, TreeModel,
+};
+use std::{cell::RefCell, rc::Rc};
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+    window.set_title("First GTK+ Program");
+    window.set_border_width(10);
+    window.set_position(gtk::WindowPosition::Center);
+    window.set_default_size(350, 70);
+
+    let combobox = gtk::ComboBox::new();
+    let renderer = CellRendererToggle::new();
+    combobox.pack_start(&renderer, true);
+    combobox.add_attribute(&renderer, "active", 0);
+
+    let renderer = CellRendererText::new();
+    combobox.pack_start(&renderer, true);
+    combobox.add_attribute(&renderer, "text", 1);
+    combobox.set_entry_text_column(1);
+
+    let model = gtk::ListStore::new(&[glib::Type::BOOL, glib::Type::STRING]);
+    let data: [(u32, &dyn ToValue); 2] = [(0, &true), (1, &"data1".to_owned())];
```

Comment:
```suggestion
    let data: [(u32, &dyn ToValue); 2] = [(0, &true), (1, &"data1")];
```
the to_owned are not needed?

Reply:
applied